shopfloor_manual_product_transfer_mobile/static/wms/src/scenario/manual_product_transfer.js

```diff
@@ -66,9 +76,11 @@ const ManualProductTransfer = {
 
             <div class="detail item-detail-card" v-if="state_is('scan_destination_location')">
                 <v-card :color="utils.colors.color_for('screen_step_done')">
-                    <v-card-title>
-                        <span class="label">Quantity = {{ quantity() }}</span>
-                    </v-card-title>
+                    <packaging-qty-picker
+                        :key="make_state_component_key(['packaging-qty-picker', 1232])"
```

Comment:
1232?? this should be a uniq value. Use the id of the record if you have one

Reply:
Fixed, everywhere + tested locally.